contrib/email-notifications/auter-notify.conf

```diff
@@ -0,0 +1,12 @@
+# Auter email notification config file
+# This file should be moved to ${SCRIPTDIR}
```

Comment:
This file should not be moved to ${SCRIPTDIR} otherwise auter will attempt to execute it. As an example, see https://github.com/rackerlabs/auter/blob/a240f29d1852526cd9b4e00141b362ed916296be/auter.yumdnfModule#L40